code/go/0chain.net/blobber/zcn.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/config"
+	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/handler"
+	"github.com/0chain/blobber/code/go/0chain.net/core/chain"
+	"github.com/0chain/blobber/code/go/0chain.net/core/common"
+	"github.com/0chain/blobber/code/go/0chain.net/core/logging"
+	"github.com/0chain/blobber/code/go/0chain.net/core/node"
+	"github.com/0chain/gosdk/zcncore"
+	"go.uber.org/zap"
+)
+
+func setupOnChain() {
+	fmt.Println("[9/10] connecting to chain	")
+
+	const ATTEMPT_DELAY = 60 * 1
+
+	// setup wallet
+	fmt.Print("	+ connect to miners: ")
+	if err := handler.WalletRegister(); err != nil {
+		fmt.Println(err.Error() + "\n")
+		panic(err)
+	}
+	fmt.Print("	[OK]\n")
+
+	// setup blobber (add or update) on the blockchain (multiple attempts)
+	for i := 1; i <= 10; i++ {
```

Comment:
Do you think we should make this number configurable?

Reply:
It should be unnecessary to config it by some one, right? it works for a long time. please just ignore it